crates/ty_python_semantic/resources/mdtest/regression/1377_iteration_count_mismatch.md

```diff
@@ -0,0 +1,149 @@
+# Iteration count mismatch for highly cyclic type vars
+
+Regression test for <https://github.com/astral-sh/ty/issues/1377>.
+
+The code is an excerpt from <https://github.com/Gobot1234/steam.py> that is minimal enough to
+trigger the iteration count mismatch bug in Salsa.
+
+<!-- expect-panic: execute: too many cycle iterations -->
```

Comment:
haha, nice feature! This might be useful to track the remaining panics in https://github.com/astral-sh/ty/issues/256